### PR TITLE
Makes the XamarinTestCloud task depreciated

### PR DESCRIPTION
**Pull request by owenniblock**
### Motivation
To encourage users to start switching over to the new AppCenterTest task.

### CODE REVIEW

File: Tasks/XamarinTestCloud/task.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 126,
+        "Minor": 127,
```
Comment (madhurig) [inline Tasks/XamarinTestCloud/task.json]: We use a convention to set it to the current VSTS sprint which is 129. Would be easier to track down changes if you followed that.

Comment (madhurig) [reply Tasks/XamarinTestCloud/task.json]: There will be a .loc json file generated when you build, please push that as well.

**Comment by madhurig** [comment]
Please commit the task.json loc file that is generated by the build

**Comment by owenniblock** [comment]
Do you mean these ones?

```
new file:   Tasks/AppCenterTest/package-lock.json
new file:   package-lock.json
```

They were the ones generated during the build.

**Comment by madhurig** [comment]
There is a [tas.loc.json](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/AppCenterTest/task.loc.json) file as well that should be generated when you build.

**Comment by owenniblock** [comment]
Done - sorry for the delay. Let me know if there's anything else :)

**Comment by madhurig** [comment]
Please exclude package.json (seems like change in whitespace) and the package-lock.json (not sure why this got added).

**Comment by owenniblock** [comment]
Better? :)

**Pull request merged**